src/index.css

```diff
@@ -88,10 +88,11 @@ body {
 }
 
 .navbar_wrapper {
```

Comment:
this class is of no use now as we removed it, we should remove it from here as well